webcompat/static/css/development/components/issue.css

```diff
@@ -156,7 +156,7 @@
 /* Link to log in */
 .wc-Issue-login {
   margin-top: 2.5em;
-  text-align:center;
+  text-align:left;
```

Comment:
Can we add a space between the prop and val here? (I know it was missing before...)